components/goffice/patches/go-math.patch

```diff
@@ -0,0 +1,15 @@
+--- goffice-0.10.35/goffice/math/go-math.c	Wed Apr 15 18:43:21 2015
++++ goffice-0.10.35/goffice/math/go-math.c.new	Thu Aug 24 09:35:18 2017
+@@ -313,7 +313,11 @@
+ {
+ 	/* What a circus!  */
+ #ifdef HAVE_FINITE
+-	return finite (x);
```

Comment:
Why can't we use finite()? Perhaps, just rearrage #ifdefs to make it less confusing